pkg/apis/application/v1alpha1/app_catalog_entry_types.go

```diff
@@ -101,6 +105,18 @@ type AppCatalogEntrySpecChart struct {
 	Icon string `json:"icon,omitempty"`
 }
 
+// +k8s:openapi-gen=true
+type AppCatalogEntrySpecRestrictions struct {
+	// ClusterSingleton is a flag for whether this chart can be installed at most once per cluster.
```

Comment:
```suggestion
	// ClusterSingleton is a flag for whether this app can be installed at most once per cluster.
```

This is the text from the spec but I wonder whether we should use app rather than chart here?

Reply:
Right, `app` is more likely suitable to the word `app catalog Entry`